src/source/uniform.rs

```diff
@@ -1,14 +1,8 @@
-use std::cmp;
-use std::time::Duration;
+use std::{cmp, time::Duration};
```

Comment:
I don't like this change, can you keep it? The two changes below are fine, only issues with this.